index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Medini Website Design</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h3 class="main_heading">My Logo</h3>
+        <img class="menubutton" src='./images/Union.svg' alt="menubutton">
+    </div>
+    <section class="container mobilepara">
+        <h1>I'm Medini<br>Glad to see<br>you!</h1>
+        <p>I'm a software developer! I can help you
+            build a product, feature or website Look
+            through some of my work and 
+            experience! If you like what you see and
+            have a project you need coded, don't
+            hesitate to contact me.</p>
+        <h4>LET'S CONNECT</h4>
+        <img src='./images/social_media_icons.svg' class="social_media_icons" alt="social media icons">
```

Comment:
- [ ] Instead of using an image for all these social icons, the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)

Reply:
I have made the ul list with all the icons in place.